src/test/java/pages/HW22/loginPage.java

```diff
@@ -0,0 +1,45 @@
+package pages.HW22;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class loginPage extends basePage {
+
+
+    WebDriver driver;
+    WebDriverWait wait;
+    //locators
+    By submitButtonLocator = By.cssSelector("[type='submit']");
+    By emailField = By.cssSelector("[type='email']");
+    By passwordField = By.cssSelector("[type='password']");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

```
    private final By submitButtonLocator = By.cssSelector("[type='submit']");
    private final By emailField = By.cssSelector("[type='email']");
    private final By passwordField = By.cssSelector("[type='password']");
```